src/validations/experience.js

```diff
@@ -0,0 +1,10 @@
+export const experienceField = (value) => {
+  let helperText
```

Comment:
Why do we need to create this variable? You can return the result.

Reply:
Removed redundant variable